### ISSUE TITLE
Change branding in REM from PetaJakarta to PetaBencana

### ISSUE THREAD

**Issue by talltom** [urbanriskmap/cognicity-rem-web#8]
- logo
- top bar redesign
- map copyright etc if needed
- internal links

**Comment by mgvarley** [comment]
Let's review more on Wednesday @talltom - do you have branding assets ready?  I have some ideas on the web structure in general - might be worth looking at applying a light framework to improve maintainability and the UX

**Comment by talltom** [comment]
Agreed @mgvarley, sounds good. Pinging @abrahamq onto this thread as we just discussed and created the https://github.com/urbanriskmap/userdesign repo (currently empty) for designers to add assets and styles etc.
